hugo/layouts/partials/header.html

```diff
@@ -24,9 +24,6 @@
                             {{ partial "svg" "go-arrow-nocircle" }}
                         </a>
                     </li>
-                    <li>
-                        <a href="http://talks.sourced.tech" target="_blank" title="Talks" data-tracked>Talks</a>
```

Comment:
This affects to mobile version. If it needs to be removed on desktop it should be done by
https://github.com/src-d/landing/pull/244/files#diff-f855ccce222c15f0148462206491ca73R90